site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -55,13 +56,14 @@ name for the `GameServerAllocation` is generated when the `GameServerAllocation`
 
 The `spec` field is the actual `GameServerAllocation` specification and it is composed as follow:
 
-- `required` is a [label selector](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) 
-   (matchLabels and/or matchExpressions) from which to choose GameServers from.
-   GameServers still have the hard requirement to be `Ready` to be allocated from
-- `preferred` is an order list of [label selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
-   out of the `required` set.
-   If the first selector is not matched, the selection attempts the second selector, and so on.
-   This is useful for things like smoke testing of new game servers. 
+- `required` is a [label selector](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
+  (matchLabels and/or matchExpressions) from which to choose GameServers from.
+  GameServers still have the hard requirement to be `Ready` to be allocated from
+- `preferred` is an ordered list of preferred
+  [label selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
+  that are _optional_ to be fulfilled, but will be searched before the `required` selector.
+  If the first selector is not matched, the selection attempts the second selector, and so on.
```

Comment:
"finally, if no preferred selectors match, the required selector is searched" might also make sense here, just like above.